static/css/paragraph.less

```diff
@@ -115,3 +115,8 @@
     margin: 20px 20%;
   }
 }
+
+img {
```

Comment:
Yup. Between images being added through WordPress and images in the template, I figured this would be best... That said, if we ever wanted to do overflowing images in any way, we'd have to go back and revise this.